homeassistant/components/apsystems/sensor.py

```diff
@@ -121,33 +118,32 @@ async def async_setup_entry(
     assert device_id
```

Comment:
Remove this line and the one above... the `device_id` can now come from the runtime_data.